### PR TITLE
Updated Deployment Guides

### PR DESCRIPTION
**Pull request by JakeDawkins**
👋 

## Changes

- updated all 3 deployment guides
- structured them with sections like
  - prerequisites
  - setting up the project
  - deploying
  - anything else like setting env variables

TODO:

* [ ] Update CHANGELOG.md with your change (include reference to issue & this PR)
* [ ] Make sure all of the significant new logic is covered by tests
* [ ] Rebase your changes on master so that they can be merged easily
* [ ] Make sure all tests and linter rules pass

### CODE REVIEW

File: docs/source/deployment/now.md [hunk 1]
```diff
@@ -25,33 +30,50 @@ Deployment to Now for Node.js apps simply requires a `package.json` file to be p
...
 ```
 
-The `now` command deploys right away and gets the server up and running on the cloud. Furthermore, running the following command will automatically start delivering reports to Apollo Engine.
+The `now` command deploys right away and attempts to start the server This specific example would fail though, due to missing environment variables. There's more info on that in the [next section](#env-variables).
+
+<h3 id="automatic-github-deploys">Automatic GitHub deployment</h3>
+
+Now supports automatic deployments from GitHub on pull requests.
+
+To set up automatic deplotments, visit [https://zeit.co/github](https://zeit.co/github) and click on the "Setup Now" button.
```
Comment (evans) [inline docs/source/deployment/now.md]: deplotments -> deployments

File: docs/source/deployment/now.md [hunk 2]
```diff
@@ -25,33 +30,50 @@ Deployment to Now for Node.js apps simply requires a `package.json` file to be p
...
+
+After signing in with GitHub, a GitHub app can be added to any account or organization. Once installed, it's possible to choose which Repositories that Now will run on, and will create new deployments on Pull Requests.
```
Comment (evans) [inline docs/source/deployment/now.md]: I'm not sure where `a GitHub app` comes from. Maybe `the Now GitHub app can ...`

Comment (evans) [reply docs/source/deployment/now.md]: The tense is a bit off here and capitalization of repositories is probably not necessary. Also, I'm not sure what installation means here. Maybe activated for an org/account or does it mean activation for a repository.

**Comment by JakeDawkins** [comment]
@evans anything else here?

**Pull request merged**